operations/network/network_management_install_guide/aruba/check_hsm.md

```diff
@@ -0,0 +1,51 @@
+# Check HSM
```

Comment:
what are we checking HSM for? I think this is part of the dhcp t-shooting confluence page?  Should we put this into a dhcp-tshooting folder?  Or make a dhcp-tshooting file?